[0m11:18:05.544615 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbe50da0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffd5e87a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbc91fa0>]}


============================== 11:18:05.555943 | 23a7abdf-bae7-4ac4-a0b9-c571341844ab ==============================
[0m11:18:05.555943 [info ] [MainThread]: Running with dbt=1.9.0
[0m11:18:05.556862 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/root/.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt init my_project', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m11:18:05.585367 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/root/.dbt')}
[0m11:18:05.586170 [info ] [MainThread]: Creating dbt configuration folder at 
[0m11:18:05.587956 [debug] [MainThread]: Starter project path: /usr/local/lib/python3.12/site-packages/dbt/include/starter_project
[0m11:18:05.600011 [info ] [MainThread]: 
Your new dbt project "my_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:18:05.600481 [info ] [MainThread]: Setting up your profile.
[0m11:18:05.602259 [error] [MainThread]: Encountered an error:

[0m11:18:05.604270 [error] [MainThread]: Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/dbt/cli/requires.py", line 153, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/cli/requires.py", line 103, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/cli/main.py", line 477, in init
    results = task.run()
              ^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/task/init.py", line 346, in run
    self.setup_profile(profile_name)
  File "/usr/local/lib/python3.12/site-packages/dbt/task/init.py", line 261, in setup_profile
    adapter = self.ask_for_adapter_choice()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/task/init.py", line 242, in ask_for_adapter_choice
    numeric_choice = click.prompt(prompt_msg, type=click.INT)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m11:18:05.606216 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": false, "command_wall_clock_time": 0.12494975, "process_in_blocks": "61984", "process_kernel_time": 0.090936, "process_mem_max_rss": "100604", "process_out_blocks": "1479", "process_user_time": 0.989307}
[0m11:18:05.606852 [debug] [MainThread]: Command `dbt init` failed at 11:18:05.606674 after 0.13 seconds
[0m11:18:05.607353 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffb731d90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffb7d5310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffb7d7140>]}
[0m11:18:05.607820 [debug] [MainThread]: Flushing usage events
[0m11:18:06.628311 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m11:19:57.026088 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbc24350>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbc27950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbc938f0>]}


============================== 11:19:57.038504 | 64867323-76e1-47b3-9265-929355e19326 ==============================
[0m11:19:57.038504 [info ] [MainThread]: Running with dbt=1.9.0
[0m11:19:57.039516 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/root/.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt init my_project_dbt', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m11:19:57.072032 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/root/.dbt')}
[0m11:19:57.072960 [info ] [MainThread]: Creating dbt configuration folder at 
[0m11:19:57.075423 [debug] [MainThread]: Starter project path: /usr/local/lib/python3.12/site-packages/dbt/include/starter_project
[0m11:19:57.089534 [info ] [MainThread]: 
Your new dbt project "my_project_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:19:57.090064 [info ] [MainThread]: Setting up your profile.
[0m11:19:57.091887 [error] [MainThread]: Encountered an error:

[0m11:19:57.094375 [error] [MainThread]: Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/dbt/cli/requires.py", line 153, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/cli/requires.py", line 103, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/cli/main.py", line 477, in init
    results = task.run()
              ^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/task/init.py", line 346, in run
    self.setup_profile(profile_name)
  File "/usr/local/lib/python3.12/site-packages/dbt/task/init.py", line 261, in setup_profile
    adapter = self.ask_for_adapter_choice()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/dbt/task/init.py", line 242, in ask_for_adapter_choice
    numeric_choice = click.prompt(prompt_msg, type=click.INT)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m11:19:57.096848 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": false, "command_wall_clock_time": 0.11781954, "process_in_blocks": "27264", "process_kernel_time": 0.106553, "process_mem_max_rss": "100924", "process_out_blocks": "1479", "process_user_time": 1.063527}
[0m11:19:57.097572 [debug] [MainThread]: Command `dbt init` failed at 11:19:57.097425 after 0.12 seconds
[0m11:19:57.098309 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffb7324e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbfaac00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffb9b1e20>]}
[0m11:19:57.098860 [debug] [MainThread]: Flushing usage events
[0m11:19:58.143793 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m11:21:42.415886 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffc162630>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbadc800>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffc437710>]}


============================== 11:21:42.425033 | 6c2068b7-c8aa-494e-9efe-1230cfe99e35 ==============================
[0m11:21:42.425033 [info ] [MainThread]: Running with dbt=1.9.0
[0m11:21:42.426064 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/root/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init my_project_dbt', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m11:21:42.433665 [info ] [MainThread]: A project called my_project_dbt already exists here.
[0m11:21:42.436070 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 0.08884341, "process_in_blocks": "28752", "process_kernel_time": 0.101284, "process_mem_max_rss": "98956", "process_out_blocks": "2", "process_user_time": 1.05032}
[0m11:21:42.436735 [debug] [MainThread]: Command `dbt init` succeeded at 11:21:42.436581 after 0.09 seconds
[0m11:21:42.437115 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffd5ec2c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbfb53a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbfb4980>]}
[0m11:21:42.437465 [debug] [MainThread]: Flushing usage events
[0m11:21:43.496886 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m11:23:10.383552 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbc858b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffba8cb60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffcb8a180>]}


============================== 11:23:10.397241 | 44b66151-d3a6-48a9-abd3-c8c84fa9f319 ==============================
[0m11:23:10.397241 [info ] [MainThread]: Running with dbt=1.9.0
[0m11:23:10.399594 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': '/root/.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m11:23:10.445358 [info ] [MainThread]: dbt version: 1.9.0
[0m11:23:10.445810 [info ] [MainThread]: python version: 3.12.4
[0m11:23:10.446486 [info ] [MainThread]: python path: /usr/local/bin/python
[0m11:23:10.447063 [info ] [MainThread]: os info: Linux-6.10.14-linuxkit-x86_64-with-glibc2.31
[0m11:23:10.447560 [info ] [MainThread]: Using profiles dir at /root/.dbt
[0m11:23:10.448113 [info ] [MainThread]: Using profiles.yml file at /root/.dbt/profiles.yml
[0m11:23:10.448542 [info ] [MainThread]: Using dbt_project.yml file at /opt/airflow/dbt/dbt_project.yml
[0m11:23:10.449014 [info ] [MainThread]: Configuration:
[0m11:23:10.449373 [info ] [MainThread]:   profiles.yml file [[31mERROR not found[0m]
[0m11:23:10.449821 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m11:23:10.450317 [info ] [MainThread]: Required dependencies:
[0m11:23:10.450863 [debug] [MainThread]: Executing "git --help"
[0m11:23:10.474330 [debug] [MainThread]: STDOUT: "b"usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone             Clone a repository into a new directory\n   init              Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add               Add file contents to the index\n   mv                Move or rename a file, a directory, or a symlink\n   restore           Restore working tree files\n   rm                Remove files from the working tree and from the index\n   sparse-checkout   Initialize and modify the sparse-checkout\n\nexamine the history and state (see also: git help revisions)\n   bisect            Use binary search to find the commit that introduced a bug\n   diff              Show changes between commits, commit and working tree, etc\n   grep              Print lines matching a pattern\n   log               Show commit logs\n   show              Show various types of objects\n   status            Show the working tree status\n\ngrow, mark and tweak your common history\n   branch            List, create, or delete branches\n   commit            Record changes to the repository\n   merge             Join two or more development histories together\n   rebase            Reapply commits on top of another base tip\n   reset             Reset current HEAD to the specified state\n   switch            Switch branches\n   tag               Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch             Download objects and refs from another repository\n   pull              Fetch from and integrate with another repository or a local branch\n   push              Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m11:23:10.475021 [debug] [MainThread]: STDERR: "b''"
[0m11:23:10.475356 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m11:23:10.475700 [info ] [MainThread]: Connection test skipped since no profile was found
[0m11:23:10.476386 [info ] [MainThread]: [31m2 checks failed:[0m
[0m11:23:10.476794 [info ] [MainThread]: dbt looked for a profiles.yml file in /root/.dbt/profiles.yml, but did
not find one. For more information on configuring your profile, consult the
documentation:

https://docs.getdbt.com/docs/configure-your-profile


[0m11:23:10.477245 [info ] [MainThread]: Project loading failed for the following reason:
 project path </opt/airflow/dbt/dbt_project.yml> not found

[0m11:23:10.479666 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.14093125, "process_in_blocks": "18640", "process_kernel_time": 0.09888, "process_mem_max_rss": "100100", "process_out_blocks": "1474", "process_user_time": 1.039252}
[0m11:23:10.480385 [debug] [MainThread]: Command `dbt debug` failed at 11:23:10.480242 after 0.14 seconds
[0m11:23:10.481201 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffb974800>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffb978710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffbd5e120>]}
[0m11:23:10.481745 [debug] [MainThread]: Flushing usage events
[0m11:23:11.506373 [debug] [MainThread]: An error was encountered while trying to flush usage events
